### PR TITLE
Add Two Dark color scheme

### PR DESCRIPTION
**Pull request by ScottKillen**
This is the two dark color scheme, based on the VS Code's One Dark theme.

2020-01-07 08_44_42-Welcome to QOwnNotes - QOwnNotes 20 1 3

### CODE REVIEW

**Comment by Waqar144** [comment]
This will replace the existing `Dark` theme.

File: src/configurations/schemes.conf [hunk 1]
```diff
@@ -1585,3 +1585,147 @@ FallbackSchema=EditorColorSchema-cdbf28fc-1ddc-4d13-bb21-6a4043316a2f
 Underline_0=true
 Underline_18=true
 Underline_26=true
+
+[EditorColorSchema-cdbf28fc-1ddc-4d13-bb21-6a4043316a2f]
```
Comment (Waqar144) [inline src/configurations/schemes.conf]: This will replace the existing Dark theme.

Comment (ScottKillen) [reply src/configurations/schemes.conf]: Sorry!

I used the UUID produced by QOWNNotes and didn't check for conflicts. I did begin with a copy of the Dark theme.

Comment (ScottKillen) [reply src/configurations/schemes.conf]: Resolved in e638c93a6554b3b38e0509dc5db88d10808d2cca

**Comment by pbek** [comment]
Great. :grin:  `FallbackSchema` is also missing, I'll add that.

**Comment by pbek** [comment]
I cannot see any difference to the `Dark` theme (even the new headline color variations are the same). Are you sure you added the right schema?

**Comment by pbek** [comment]
I'll revert for the next release.

**Pull request merged**